crates/holochain/src/local_network_tests.rs

```diff
@@ -35,6 +35,7 @@ use test_case::test_case;
 #[test_case(2)]
 #[test_case(4)]
 #[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
```

Comment:
i'll look here

Reply:
This one is already in hand, I know what's going on with it and have a PR up that helps reveal the problem